exotic/exotic.py

```diff
@@ -2280,8 +2357,7 @@ def main():
                             'a2': 0,             #Flux lower bound
                         }
 
-                        # phase = (arrayTimes[~filtered_data]-prior['tmid'])/prior['per']
-                        phase = getPhase(arrayTimes[~filtered_data], prior['per'], prior['tmid'])
+                        phase = (arrayTimes[~filtered_data]-prior['tmid'])/prior['per']
```

Comment:
why not call the function instead of redefining things here?

Reply:
The function returns a phase between -1 and 1. I need the orbital phase with the integer in front of it indicating the orbital epoch. The equation below that line should match your paper equation for predicting transits...